### PR TITLE
[SONAR] Cleanup

### PR DESCRIPTION
**Pull request by hansva**
cleanup

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:
- [ ] Run `mvn clean install apache-rat:check` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
- [ ] If you have a group of commits related to the same change, please squash your commits into one and force push your branch using `git rebase -i`.
- [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable.

To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

- [ ] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
- [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: core/src/main/java/org/apache/hop/core/DbCacheEntry.java [hunk 1]
```diff
@@ -69,8 +69,8 @@ public int hashCode() {
 
   @Override
   public boolean equals(Object o) {
-    if (o instanceof DbCacheEntry) {
-      DbCacheEntry obj = (DbCacheEntry) o;
+    if (o instanceof DbCacheEntry dbCacheEntry) {
+      DbCacheEntry obj = dbCacheEntry;
```
Comment (nadment) [inline core/src/main/java/org/apache/hop/core/DbCacheEntry.java:73]: Can be simplified

File: core/src/main/java/org/apache/hop/core/row/value/timestamp/SimpleTimestampFormat.java [hunk 2]
```diff
@@ -243,8 +243,8 @@ public StringBuffer format(Date timestamp, StringBuffer toAppendTo, FieldPositio
     StringBuffer dateBuffer;
     String nan;
 
-    if (timestamp instanceof Timestamp) {
-      Timestamp tmp = (Timestamp) timestamp;
+    if (timestamp instanceof Timestamp timestamp1) {
+      Timestamp tmp = timestamp1;
```
Comment (nadment) [inline core/src/main/java/org/apache/hop/core/row/value/timestamp/SimpleTimestampFormat.java:247]: Can be simplified

File: core/src/main/java/org/apache/hop/core/logging/Slf4jLoggingEventListener.java [hunk 3]
```diff
@@ -54,8 +54,8 @@ public Slf4jLoggingEventListener() {
   public void eventAdded(HopLoggingEvent event) {
     Object messageObject = event.getMessage();
     checkNotNull(messageObject, "Expected log message to be defined.");
-    if (messageObject instanceof LogMessage) {
-      LogMessage message = (LogMessage) messageObject;
+    if (messageObject instanceof LogMessage logMessage) {
+      LogMessage message = logMessage;
```
Comment (nadment) [inline core/src/main/java/org/apache/hop/core/logging/Slf4jLoggingEventListener.java:58]: Can be simplified

**Comment by hansva** [comment]
you are right, I'll do that. It was some low brain activity cleanup and I did not really "look" at the code was just focused on cleaning up the warnings.

**Comment by hansva** [comment]
I think I have them all now

**Pull request merged**